Header   header
string   id         # network ssid
uint8[6] bssid      # base station mac address
string   dev        # wlan device for this scan
float32  frequency  # channel which was scanned
float32  rssi       # rssi value in dBm
